bindings/java/pom.xml

```diff
@@ -32,7 +32,7 @@
 
     <groupId>org.apache.opendal</groupId>
     <artifactId>opendal-java</artifactId>
-    <version>0.45.1</version>
+    <version>0.46.0</version>
```

Comment:
No breaking changes in java, we just need to release 0.45.2

Reply:
Let's do a simultaneous release for Java binding, and perhaps Node.js one.

Before 1.0, the version doesn't tell to many details. And I don't want to keep a unaligned version for core and java binding because users should just bump to the latest version and if they don't find a breaking change, all goes well.